docs/api/Query.md

```
@@ -110,6 +110,10 @@ const query = new Query(queryClient, () => ({
 console.log(query.data);
 ```
 
+::: tip Mutating `data` does not trigger MobX updates by default
+[`resultObservable`](#resultobservable-queryfeature) defaults to **`'ref'`**, so MobX reacts when **`data`** is replaced, not when you change fields on the same object. Use **`'deep'`** on the query or in [`QueryClient` defaults](/api/QueryClient#queryfeatures), or keep **`'ref'`** and update with [`setData`](#setdata-updater-options) returning `{ ...previous, ...updates }`.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)